sdk/src/test/kotlin/org/ocast/sdk/discovery/DeviceDiscoveryTest.kt

```diff
@@ -168,6 +169,53 @@ class DeviceDiscoveryTest {
         val removedDevicesCaptor = argumentCaptor<List<UpnpDevice>>()
         verify(listener, times(1)).onDevicesRemoved(removedDevicesCaptor.capture())
         assertEquals(listOf(firstDevice), removedDevicesCaptor.firstValue)
+        verify(listener, never()).onDevicesChanged(any())
+        verify(listener, never()).onDiscoveryStopped(anyOrNull())
+    }
+
+    @Test
+    fun receiveChangedDeviceDescriptionCallsListenerOnDeviceChanged() {
+        // Given
+        val oldSearchResponseString = "HTTP/1.1 200 OK\r\n" +
```

Comment:
`M` is missing in variable name.
This should be: `oldMSearchResponseString` (same for `newSearchResponseString`).